patches/minecraft/net/minecraft/world/entity/Entity.java.patch

```diff
@@ -595,7 +651,7 @@
 +   /**
 +    * Accessor method for {@link #getEyeHeight(Pose, EntityDimensions)}
 +    */
-+   public float getEyeHeightAccess(Pose pose, EntityDimensions size) {
++   public final float getEyeHeightAccess(Pose pose, EntityDimensions size) {
 +      return this.m_6380_(pose, size);
 +   }
 +
```

Comment:
Given that `getEyeHeight(Pose, Dims)` is deprecated now, this method can probably be removed, since it should not be being used externally in favor of the event-computed value (which is accessible via `getEyeHeight()`)

Reply:
This accessor is needed because `getEyeHeight(Pose, EntityDimensions)` is protected and can't be called in `IForgeEntity#getEyeHeightForge(Pose, EntityDimensions)`.